perftools/stresstesting/login-and-learn.yml

```diff
@@ -0,0 +1,22 @@
+config:
```

Comment:
In KA Lite, all these tools quickly became dysfunctional - they were one-timers. I think this might from the beginning sit better in a separate repo to not clutter the main Kolibri repo? Is this very annoying? It potentially just means that you can maintain the benchmarking tools separately and with a Kolibri version dependency explicitly specified.